monai/data/image_reader.py

```diff
@@ -833,8 +834,13 @@ def get_data(
         metadata["spatial_shape"] = np.asarray(region.shape[:-1])
         metadata["original_channel_dim"] = -1
 
+        # combine image and metadata
+        if not isinstance(region, MetaTensor):
+            region = MetaTensor(region, meta=metadata)
+
         # Make it channel first
-        region = EnsureChannelFirst()(region, metadata)
+        region = EnsureChannelFirst()(region).numpy()
```

Comment:
And may I know why cocnvert it to numpy array?

Thanks.

Reply:
Prior to these changes, it was a `np.ndarray` that gets passed through. Passing a `torch.Tensor` causes knock-on problems that I didn't think were worth spending the time to fix.

Reply:
OK, I see your point, I think it's OK for the current stage, could you please help add a `FIXME` comment?

Thanks!

Reply:
Done.